portfolio/pom.xml

```diff
@@ -37,6 +42,7 @@
           <!-- TODO: set project ID. -->
           <deploy.projectId>abakir-step-2020</deploy.projectId>
           <deploy.version>1</deploy.version>
+          <automaticRestart>true</automaticRestart>
```

Comment:
This doesn't seem to align with the scope of this pull request.

Reply:
I was trying to find a way when I refresh the page, my updated code shows, but I am still working on that

Reply:
This is still outside of the scope for this change. I'd suggest moving it to a different pull request if you want to commit it to this remote repository.

Reply:
okay I will do that next time

Reply:
Ok. Just send another review once you've removed it from here, so I can take another look.

Reply:
Okay just fixed it

Reply:
okay just fixed it